py/desispec/sky.py

```diff
@@ -655,6 +666,104 @@ def fit_and_interpolate(delta,skyfibers,mean,components,label="") :
     cskyflux[:,bad]=0.
     modified_cskyivar[:,bad]=0.
 
+    if fit_offsets :
+
+        cfinder = CalibFinder([frame.meta])
+        amps    = get_amp_ids(frame.meta)
+
+        psf_filename = findfile('psf',frame.meta["NIGHT"],frame.meta["EXPID"],frame.meta["CAMERA"])
+        if not os.path.isfile(psf_filename) :
+            log.error("No PSF file "+psf_filename)
+            raise IOError("No PSF file "+psf_filename)
+        log.info("Using PSF {}".format(psf_filename))
+        tset = read_xytraceset(psf_filename)
+
+        tmp_fibers = np.arange(frame.nspec)
+        tmp_x = np.zeros(frame.flux.shape,dtype=float)
+        tmp_y = np.zeros(frame.flux.shape,dtype=float)
+        for fiber in tmp_fibers :
+            tmp_x[fiber] = tset.x_vs_wave(fiber=fiber,wavelength=frame.wave)
+            tmp_y[fiber] = tset.y_vs_wave(fiber=fiber,wavelength=frame.wave)
```

Comment:
Apologies as I read desispec code for the first time.  Does this duplicate lines 714+ below and not actually get used?

Reply:
yes. thank you.